packages/xs-extra/xapi-expiry-alerts.master/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "xen-api@lists.xen.org"
+authors: [ "xen-api@lists.xen.org" ]
+homepage: "https://github.com/xapi-project/xen-api"
+bug-reports: "https://github.com/xapi-project/xen-api/issues"
+dev-repo: "git+https://github.com/xapi-project/xen-api.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs ]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml"
+  "dune" {build & >= "1.4"}
+  "xapi-client"
+  "xapi-consts"
+  "xapi-types"
+  "xapi-stdext-date"
+]
+synopsis: "Library required by xapi"
```

Comment:
This needs a better description.

Reply:
Please also run `opam lint --normalise opam`

Reply:
OK, updated